tools/hidl/hidlcomp.cpp

```diff
@@ -5760,7 +5751,7 @@ void EspServInfo::write_esp_binding()
         {
             outf("\tprofileOptions = std::string(\"%s\");\n", mthi->getExecutionProfilingOptions().c_str());
             outs("\tbuckets = getExecutionProfileBuckets(profileOptions);\n");
-            outf("\t%s = hpccMetrics::registerCyclesToNsScaledHistogramMetric(\"%s\", \"\", buckets);\n", mthi->getExecutionProflingMetricName().c_str(), mthi->getName());
+            outf("\t%s = hpccMetrics::registerCyclesToNsScaledHistogramMetric(\"%s\", \"\", buckets);\n", mthi->getExecutionProfilingMetricName().c_str(), mthi->getName());
```

Comment:
Missed the comment above about combining the functions - possibly because it wasn't attached to a line that has changed.

Reply:
Expanded the comment above the loop to indicate that its initializing the histogram metric for each method with execution profiling enabled.